docs/pages/build-reference/variables.md

```diff
@@ -61,9 +61,108 @@ See the [eas.json reference](/build/eas-json.md) for more information.
 
 ## Using secrets in environment variables
 
-🚧 We are currently working on a secrets API that will allow developers to store generic encrypted secrets and selectively expose them to build jobs. This feature will be available before EAS Build graduates from preview.
+For values that you want to expose to your EAS Build hooks but are too sensitive to include in your source code, you can use our Secrets feature found on the Expo website.
 
-App signing credentials secrets are stored in `credentials.json`, which should not be committed to git. This file can also be used to set the `NPM_TOKEN` environment variable in order to give you access to your organization's private packages. [Learn more](how-tos.md).
+These secrets are encrypted at rest and in transit, and are only decrypted in a secure environment by EAS servers.
+
+You can create up to 100 account-wide and app-specific secrets. Account-wide, or "global" secrets will be exposed to every build environment across all of your apps. App-specific secrets only apply to the app they're defined on, and override global secrets with the same name.
+
+For managing secrets, we provide a web-based UI and a set of EAS CLI commands.
+
+### Finding the secrets web UI
+
+To create secrets to use across all apps in an account, you can navigate to the "Secrets" tab under the account settings:
+
+![Global secrets location](/static/images/eas-build/environment-secrets/secrets-account-nav.png)
+
+To create secrets to in a specific app, you can navigate to the "Secrets" tab under the app dashboard:
```

Comment:
```suggestion
To create secrets to in a specific app, navigate to the "Secrets" tab under the app dashboard:
```

Reply:
it would be cool if there was a way to link to some tab without specifying which app, and when the user follows the link they're presented with a list of apps that they have access to, and upon selecting the project it will go to the appropriate tab